node/src/OpenTelemetry.ts

```diff
@@ -83,4 +83,41 @@ export class OpenTelemetry {
         InitOpenTelemetry(openTelemetryConfig);
         this._instance = new OpenTelemetry();
     }
+
+    /**
+     * Check if the OpenTelemetry instance is initialized
+     * @returns True if the OpenTelemetry instance is initialized, false otherwise
+     */
+    public static isInitialized() {
+        return this._instance != null;
+    }
+
+    /**
+     * Get the sample percentage for traces
+     * @returns The sample percentage for traces only if OpenTelemetry is initialized and the traces config is set, otherwise undefined.
+     */
+    public static getSamplePercentage() {
+        return this.openTelemetryConfig?.traces?.samplePercentage;
+    }
+
+    /**
+     * Set the percentage of requests to be sampled and traced. Must be a value between 0 and 100.
+     * This setting only affects traces, not metrics.
+     * @param percentage - The sample percentage 0-100
+     * @throws Error if OpenTelemetry is not initialized or traces config is not set
+     * @remarks
+     * This method can be called at runtime to change the sampling percentage without reinitializing OpenTelemetry.
+     */
+    public static setSamplePercentage(percentage: number) {
+        if (!this.openTelemetryConfig || !this.openTelemetryConfig.traces) {
+            Logger.log(
+                "error",
+                "GlideOpenTelemetry",
+                "OpenTelemetry config traces not initialized",
+            );
+            throw new Error("OpenTelemetry config traces not initialized");
```

Comment:
we should raise glide error and not a general one - maybe something like config error